torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10861,6 +10867,8 @@ def ref_pairwise_distance(input1, input2):
                # RuntimeError: !lhs.isAliasOf(rhs)INTERNAL ASSERT FAILED at
                # "../torch/csrc/jit/passes/utils/check_alias_annotation.cpp":104, please report a bug to PyTorch.
                DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
+               DecorateInfo(unittest.skip("Skipped! RuntimeError: bias tensor has to be contiguous"), 'TestGradients',
```

Comment:
Should we just add a `.contiguous()` in the formula to fix this?
This can be in a follow up PR

Reply:
Yeah and maybe with this?
```
#if defined(USE_ROCM)
```